### PR TITLE
Feature/1561 change logging level during build

### LINKED ISSUE #1561: Monitor-ui integration tests log a lot of debug messages during build
When doing a maven build of DataCleaner (or just the datacleaner-monitor-ui module) using the integration-test profile (e.g. `mvn clean verify -Pintegration-test`), the integration tests of the datacleaner-monitor-ui module are run and they produce lots and lots of debug messages. 

We should see if it is possible to, by default, disable those debug messages.

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Fixes #1561 

The logging level for IT tests has been changed from 'INFO' to 'WARN'
image

### CODE REVIEW

**Comment by arjansh** [comment]
Uh, I actually can't reproduce the original issue.

**Comment by ClaudiaPHI** [comment]
I saw it, It write everything on the screen: the job, the call and the response.

File: monitor/ui/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	</appender>
+
+	<logger name="org.datacleaner.monitor" level="warn" />
+	<logger name="org.datacleaner" level="warn" />
+	<logger name="org.datacleaner.descriptors.DCClassVisitor" level="error" />
+
+	<root level="warn">
+		<appender-ref ref="consoleAppender" />
+	</root>
+</configuration>
```
Comment (LosD) [inline monitor/ui/src/test/resources/logback-test.xml]: Missing EOL

**Comment by LosD** [comment]
That's weird, mine also stopped (on master).

**Comment by arjansh** [comment]
@ClaudiaPHI I'm not sure what the sreenshot is telling us.

**Comment by LosD** [comment]
@arjansh I'm pretty sure the idea is to show that there's no logging noise :)

**Comment by ClaudiaPHI** [comment]
@jhorcicka , @LosD mergeable ?

**Comment by LosD** [comment]
Fine by me. @arjansh any objections?

**Comment by arjansh** [comment]
@LosD No objections.

**Pull request merged**